tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/ll.set2.llcp.test_list

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Oticon A/S
+# SPDX-License-Identifier: Apache-2.0
+
+LL/CON/SLA/BV-29-C
+LL/CON/SLA/BV-33-C
+LL/CON/SLA/BV-34-C
+LL/CON/SLA/BV-40-C
+LL/CON/SLA/BV-42-C
+#LL/CON/SLA/BV-77-C # requires update in EDTT due to change in DLE algorithm
+#LL/CON/SLA/BV-78-C # requires update in EDTT due to change in DLE algorithm
+#LL/CON/SLA/BV-80-C # requires update in EDTT due to change in DLE algorithm
+#LL/CON/SLA/BV-81-C # requires update in EDTT due to change in DLE algorithm
+LL/DDI/ADV/BV-01-C
+LL/DDI/ADV/BV-02-C
+LL/DDI/ADV/BV-03-C
+LL/DDI/ADV/BV-04-C
+LL/DDI/ADV/BV-05-C
+LL/DDI/ADV/BV-06-C
+LL/DDI/ADV/BV-07-C
+LL/DDI/ADV/BV-08-C
+LL/DDI/ADV/BV-09-C
+LL/DDI/ADV/BV-11-C
+LL/DDI/ADV/BV-15-C
+LL/DDI/ADV/BV-16-C
+LL/DDI/ADV/BV-17-C
+LL/DDI/ADV/BV-18-C
+LL/DDI/ADV/BV-19-C
+LL/DDI/ADV/BV-20-C
+LL/DDI/SCN/BV-01-C
+LL/DDI/SCN/BV-02-C
+LL/DDI/SCN/BV-03-C
+LL/DDI/SCN/BV-04-C
+LL/DDI/SCN/BV-05-C
+LL/DDI/SCN/BV-10-C
+LL/DDI/SCN/BV-11-C
+LL/DDI/SCN/BV-12-C
+LL/DDI/SCN/BV-13-C
+LL/DDI/SCN/BV-14-C
+LL/DDI/SCN/BV-15-C
+LL/DDI/SCN/BV-16-C
+LL/DDI/SCN/BV-17-C
+LL/DDI/SCN/BV-18-C
+LL/DDI/SCN/BV-26-C
+LL/DDI/SCN/BV-28-C
+LL/SEC/ADV/BV-02-C
+LL/SEC/ADV/BV-03-C
+LL/SEC/ADV/BV-04-C
+LL/SEC/ADV/BV-05-C
+LL/SEC/ADV/BV-06-C
+LL/SEC/ADV/BV-08-C
+LL/SEC/ADV/BV-09-C
+LL/SEC/ADV/BV-10-C
+LL/SEC/ADV/BV-11-C
+LL/SEC/ADV/BV-12-C
+LL/SEC/ADV/BV-13-C
+LL/SEC/ADV/BV-14-C
+LL/SEC/ADV/BV-15-C
+LL/SEC/ADV/BV-16-C
+LL/SEC/ADV/BV-17-C
+LL/SEC/ADV/BV-18-C
+LL/SEC/ADV/BV-20-C
+LL/SEC/SCN/BV-01-C
```

Comment:
Are these additional tests compare to being tested for legacy LLCP?

Reply:
same tests as for legacy but for new LLCP